app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(person_data_list: list) -> list:
+    person_objects = []
+
+    for person_data in person_data_list:
+        person_obj = Person(person_data["name"], person_data["age"])
+        person_objects.append(person_obj)
```

Comment:
It would be more clear to use use list comprehension

Reply:
Done